src/jdk.compiler/share/classes/com/sun/tools/javac/code/Lint.java

```diff
@@ -476,27 +473,6 @@ public boolean isSuppressed(LintCategory lc) {
         return suppressedValues.contains(lc);
     }
 
-    /**
-     * Helper method. Log a lint warning if its lint category is enabled.
-     *
-     * @param warning key for the localized warning message
-     */
-    public void logIfEnabled(LintWarning warning) {
```

Comment:
I like it a lot that now there's "just warnings". It took a while to see this more clearly, but once you see it you can't unsee it :-)